### DISCUSSION TITLE
The vweb is running with an error.

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#19019] [Questions and Answers]
![image](https://github.com/vlang/v/assets/35259040/76b72917-eda2-452f-acd4-b5ee79419239)

--
Commenting out a portion of the code allows it to run normally.
![image](https://github.com/vlang/v/assets/35259040/3903a836-209d-49dd-a2f8-23a790c85e27)

**Comment by JalonSolov** [comment]
When you post screenshots, all we can do is go "Ooo... look at the pretty colors...".

If you want help with something, you need to paste source for a complete example.  And it would be better to open that as an issue, not a discussion.
